pom.xml

```diff
@@ -41,7 +41,7 @@
   <description>
     This plugin makes CloudBees DevOptics components available in Jenkins.
   </description>
-  <url>https://wiki.jenkins.io/display/JENKINS/CloudBees+DevOptics+Enabler+Plugin</url>
+  <url>https://wiki.jenkins.io/display/JENKINS/CloudBees+DevOptics+Installation+Plugin</url>
```

Comment:
@jordanglassman we should bump `jenkins.version` to `2.73.3` to align with the minimum plugin version of DevOptics while we are at it